lib/fs-loader.js

```diff
@@ -70,60 +70,64 @@ const loadModule = async (dir, options) => {
 
 // Creates Metaschema object and fills it with schemas
 //   dir <string> | <string[]>
-//   loadOptions <Object>
-//     context
-//     decorators
+//   options <Object>
+//     context <Object>
+//     decorators <Object>
 //     localDecorators <Object>, optional
 //       [type] <Object> decorators specific for this schema type
 //     pathToType <Object>
 //       [ext] <string>
 //     pathToType <Function>
 //       path <string>
 //     Returns: <string>
+//     loadOrder <Function>, optional
```

Comment:
There is no such argument.

Reply:
https://github.com/metarhia/metaschema/blob/8162a20cb76fea0777926298c15667bedb4a047d/lib/config.js#L130

Reply:
Remove it, it's not used anywhere.